examples/box.jl

```diff
@@ -202,38 +200,50 @@ end
 ##       Output: state_new = [T1(t+dt), ..., S3(t+dt)]
 
 function compute_update(state_now, state_old, u, params, dt)
-
     dstate_now_dt = zeros(6)
     state_new = zeros(6)
 
     ## first computing the time derivatives of the various temperatures and salinities
-    if u > 0 
-
-        dstate_now_dt[1] = u * (state_now[3] - state_now[1]) / params.boxvol[1] + params.gamma * (params.Tstar[1] - state_now[1]) 
-        dstate_now_dt[2] = u * (state_now[1] - state_now[2]) / params.boxvol[2] + params.gamma * (params.Tstar[2] - state_now[2])
-        dstate_now_dt[3] = u * (state_now[2] - state_now[3]) / params.boxvol[3] 
-
-        dstate_now_dt[4] = u * (state_now[6] - state_now[4]) / params.boxvol[1] + params.FW[1] / params.boxvol[1]
-        dstate_now_dt[5] = u * (state_now[4] - state_now[5]) / params.boxvol[2] + params.FW[2] / params.boxvol[2]
+    if u > 0
```

Comment:
I find this a lot less readable,

Generally I am not the biggest fan of the Blue style.

Reply:
We can choose another one, I don't have strong preferences. SciML is better?

Reply:
YAS or Default.

Reply:
Ok, but all of these styles will break the lines as well. I think this is more of a problem with applying the formatting style post-hoc, because adhering to a style would push one to write things differently in the first place. Here, I would have abbreviated the variable names according to the math equations in the comments, i.e. use `T` instead of `state` etc.